tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -18,6 +18,14 @@
 log = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def setup_fixture(install_logging):
+    """
+    Installs openshift-logging
+    """
+    logging.info("Testcases execution post deployment of openshift-logging")
+
+
```

Comment:
Please call the setup fixture

Reply:
Yeah, I missed this `@pytest.mark.usefixtures(setup_fixture.__name__)`, let me update the code.